composer.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "pluginkollektiv/cachify",
+  "type": "wordpress-plugin",
+  "license": "GPL 2.0",
```

Comment:
Actually Cachify is licensed unter GPLv2 **or later**, so according to SPDX, the correct license string should be `GPL-2.0+`. Even if we'd omit the +, it should still be `GPL-2.0`.